### ISSUE TITLE
follow_mouse_focus missing feature/bug with multi-monitor

### ISSUE THREAD

**Issue by Crystal4276** [qtile/qtile#3993]
### The issue:

qtile-git
qtile extra-git
Three monitors setup : screens [1,0,2]
the video below only show screen 1(left) and 0 (right) for the sake of clarity.
I'm using in my config file  'follow_mouse_focus=True'

I wanted qtile to follow mouse focus when going from screen 1 to 0.
This is not happening. It's illustrated in video as when i open a new application, the latter appears on screen 1.

https://user-images.githubusercontent.com/27446196/201617070-c2c07599-2671-43cc-b849-9a08b6157165.mp4

`follow_mouse_focus` only works with multi-monitor when a window is displayed on the screen.
Could this behavior tuned to include screen changes ?

### Required:

- [X] I have searched past issues to see if this bug has already been reported.

**Comment by Crystal4276** [comment]
Confirming it's still there on 0.23 version

**Comment by elParaguayo** [comment]
Yeah - I should pick this up as the upstream fixes I mentioned earlier have been implemented which means I could get this to work (at least on x11. Wayland is a mystery to me...)
